metals/src/main/scala/scala/meta/internal/metals/WorkspaceLspService.scala

```diff
@@ -151,10 +156,33 @@ class WorkspaceLspService(
 
   private val timerProvider: TimerProvider = new TimerProvider(time)
 
+  def getHttpServer(): Future[Option[MetalsHttpServer]] = {
+    httpServer match {
+      // If there is no doctor format set then no doctor will be available otherwise
+      case HttpServerOff if clientConfig.doctorFormat() == None =>
```

Comment:
What about:
```Scala
case HttpServerOff if clientConfig.doctorFormat() != None =>
```
Also how can `clientConfig.doctorFormat() == None`? It has a `getOrElse` part. Shouldn't it be something in line of `clientConfig.isHttpEnabled() == false`?

Reply:
Ach right. I changed it to use isExecuteClientCommandProvider, since if both that is disabled and no server is running then user might want to start it.